builtin/logical/pki/cert_util.go

```diff
@@ -745,11 +745,10 @@ func signCert(b *backend,
 
 	csrString := data.apiData.Get("csr").(string)
 	if csrString == "" {
-		return nil, errutil.UserError{Err: fmt.Sprintf("\"csr\" is empty")}
+		return nil, errutil.UserError{Err: "\"csr\" is empty"}
 	}
 
-	pemBytes := []byte(csrString)
-	pemBlock, pemBytes := pem.Decode(pemBytes)
```

Comment:
We really aught to get more consistent in our handling of this. _Ideally_ we'd validate that there's no trailing (non-whitespace) data after PEM blocks if we only process one...